src/ontology/mondo-edit.obo

```diff
@@ -619455,6 +619461,15 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4585 {source="PMID:27683935"} ! GRIN2A
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9063" xsd:anyURI
 
+[Term]
+id: MONDO:1060149
+name: ACAN-related short stature spectrum
+def: "A rare semidominant genetic skeletal disorder characterized by short stature with variable phenotypic features which may include osteochondritis dissecans, advanced bone age, early-onset arthritis, and/or features consistent with spondyloepiphyseal dysplasia, Kimberley type caused by a single allele whereas biallelic variation can cause spondyloepimetaphyseal dysplasia, aggrecan type." [PMID:27311816, PMID:35434101]
```

Comment:
I feel like "a variation in ACAN" should be included in the definition (based on the name). 
Also, maybe the information about the "Kimberley type" and the "spondyloepimetaphyseal dyslasia aggrecan type" should be added to the corresponding terms (either in the definition or in the comment, wherever it is most appropriate for you).